src/Microsoft.InsightsGenerator/Workflow.cs

```diff
@@ -0,0 +1,105 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.InsightsGenerator
+{
+    using System;

```

Comment:
could you please put all the using together at the top of the file and sort them?  that's the style convention we've been using so far in the repo, so it would be good to be consistent.

Reply:
you mean outside the namespace and sorted right?

Reply:
done